startup.py

```diff
@@ -13,6 +13,11 @@
 import shutil
 import hashlib
 import socket
+import ctypes.wintypes
```

Comment:
Would you mind moving this ctypes.wintypes import to the section where sys platform is tested for win32, please ? Otherwise in linux systems this import would raise an error.